package.json

```diff
@@ -1,14 +1,31 @@
 {
-  "name": "safe-event-emitter",
+  "name": "@metamask/safe-event-emitter",
```

Comment:
Interesting choice. Generally I think anything closely tied to either of our two products should go in our org, but I'm less sure of cases like this, where the module is very applicable to other projects, and not really tied to MetaMask at all.

Reply:
I'm guessing this is why you didn't bother with a major version bump either

Reply:
I think everything should be scoped going forward, there's no harm in it

Reply:
A version bump PR is coming, next

Reply:
Well there is _some_ harm. Renaming the package adds some friction for people using it. Scoping it under `MetaMask` might make people think it's a part of `MetaMask` somehow, rather than being a general-purpose module.

:man_shrugging: After considering this further, I don't really care about those problems and I agree we should scope it.